### PR TITLE
Automate Edit Profile screen

### PR DESCRIPTION
**Pull request by bilalawan321**

### CODE REVIEW

**Comment by bilalawan321** [comment]
@jawad-khan can you please review this PR?

File: tests/android/pages/android_edit_profile.py [hunk 1]
```diff
@@ -0,0 +1,336 @@
+"""
+    Edit Profile Module
+"""
+from tests.android.pages import android_elements
+from tests.android.pages.android_base_page import AndroidBasePage
+
+
+class AndroidEditProfile(AndroidBasePage):
+    """
+    User Account screen
```
Comment (jawad-khan) [inline tests/android/pages/android_edit_profile.py:10]: Can be more descriptive.

File: tests/android/pages/android_edit_profile.py [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+        )[1]
+
+        self.global_contents.scroll_screen(self.driver, first_elem, last_elem)
+
+        all_elem = self.global_contents.get_all_views_on_screen(
+            self.driver,
+            android_elements.edit_profile_change_birth_year
+        )
+
+        for item in all_elem:
```
Comment (jawad-khan) [inline tests/android/pages/android_edit_profile.py:311]: for item in items or for element in elements sounds more logical.

**Review by jawad-khan — COMMENTED** [review]
Can we avoid code repitition?

File: tests/android/pages/android_edit_profile.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/pages/android_edit_profile.py]: fixed

File: tests/android/pages/android_edit_profile.py (hunk 2 above)
Comment (bilalawan321) [reply tests/android/pages/android_edit_profile.py]: fixed

**Review by jawad-khan — APPROVED** [review]

**Pull request merged**